[2021-12-13 13:29:07,535] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.merge_process manual__2021-12-13T13:28:54.949355+00:00 [queued]>
[2021-12-13 13:29:07,549] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.merge_process manual__2021-12-13T13:28:54.949355+00:00 [queued]>
[2021-12-13 13:29:07,549] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 13:29:07,550] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-13 13:29:07,550] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 13:29:07,567] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): merge_process> on 2021-12-13 13:28:54.949355+00:00
[2021-12-13 13:29:07,573] {standard_task_runner.py:52} INFO - Started process 6391 to run task
[2021-12-13 13:29:07,579] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'merge_process', 'manual__2021-12-13T13:28:54.949355+00:00', '--job-id', '1105', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpwli8zvn5', '--error-file', '/tmp/tmp6flhy1mp']
[2021-12-13 13:29:07,581] {standard_task_runner.py:77} INFO - Job 1105: Subtask merge_process
[2021-12-13 13:29:07,643] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.merge_process manual__2021-12-13T13:28:54.949355+00:00 [running]> on host 218df5ddc363
[2021-12-13 13:29:07,740] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=merge_process
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T13:28:54.949355+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T13:28:54.949355+00:00
[2021-12-13 13:29:07,874] {python.py:152} INFO - Done. Returned value was: None
[2021-12-13 13:29:07,895] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=merge_process, execution_date=20211213T132854, start_date=20211213T132907, end_date=20211213T132907
[2021-12-13 13:29:07,955] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-13 13:29:08,026] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
